wp-content/themes/theme-publicsource/functions.php

```diff
@@ -144,3 +144,23 @@ function publicsource_widget_custom_fields_form( $widget, $args, $instance ) {
 	remove_action( 'in_widget_form', 'largo_widget_custom_fields_form', 1 );
 	add_action( 'in_widget_form', 'publicsource_widget_custom_fields_form', 1, 3 );
 });
+
+/**
+ * Adds the ability to include a widget area in the header of a category archive page.
+ * The widget area must use the same name as the slug of the category it is going to be used on.
+ * 
+ * If a widget area with a matching name does not exist or is empty, no widget area will be displayed.
+ * 
+ * @see https://github.com/INN/umbrella-mwcir/issues/42
```

Comment:
Might want to add a link to this PR as well.

Reply:
Added in https://github.com/INN/umbrella-publicsource/pull/40/commits/e58c234db90ef2eff1a09ef161f70c0f12860ce1